Source/Scene/GltfLoader.js

```diff
@@ -188,6 +190,13 @@ Object.defineProperties(GltfLoader.prototype, {
  * @private
  */
 GltfLoader.prototype.load = function () {
+  // Skip loading glTF JSON if it was passed in as an argument.
+  if (defined(this._gltfJson)) {
+    this._state = GltfLoaderState.LOADED;
+    this._textureState = GltfLoaderState.LOADED;
+    return;
+  }
+
```

Comment:
`GltfJsonLoader` can't be skipped, there's a lot of work it does besides fetching the glTF, like loading external resources and decoding data uris.

Reply:
The JSON will need to be passed through similar to how `typedArray` is passed through. Hopefully pretty straightforward using `typedArray` as a reference.